### ISSUE TITLE
Updating an existing server with a firewall deactivates and deletes backups

### ISSUE THREAD

**Issue by agnf** [ansible-collections/hetzner.hcloud#87]
##### SUMMARY
With hetzner.hcloud.hcloud_firewall I create a new firewall. Then I attach it with hetzner.hcloud.hcloud_server to a server. During the process of attaching the server to the firewall the backup functionality is disabled and existing backups are deleted. This can be seen in the Hcloud activities log as well.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

hcloud_server.py 

##### STEPS TO REPRODUCE

```yaml
- name: Firewall for "{{inventory_hostname}}"
  hetzner.hcloud.hcloud_firewall:
    api_token: "{{ HCLOUD_TOKEN }}"
    name: "{{ inventory_hostname }}"
    state: present
    rules: "{{ fw_rules }}"
  delegate_to: localhost

- name: Attach server to firewall
  hetzner.hcloud.hcloud_server:
    api_token: "{{ HCLOUD_TOKEN }}"
    name: "{{ inventory_hostname }}"
    firewalls:
      - "{{ inventory_hostname }}"
    state: present
  delegate_to: localhost
```

##### EXPECTED RESULTS

Server is added to Firewall with no additional changes to the server.

##### ACTUAL RESULTS

At least backups get deactivated for the added server. During deactivation all backups are deleted. Currently, I can't say if any other parameters of the server also get changed or are affected. But at least backups are gone when running the playbook.

**Comment by agnf** [comment]
Upon closer inspection of the [hcloud_server documentation](https://docs.ansible.com/ansible/latest/collections/hetzner/hcloud/hcloud_server_module.html) I noticed that `backups` has a default value of `no`. This is probably the reason why my Ansible run removed the activated backup option and the backups. Which in turn would me it's case of "works as intended", although it's a thing that could be better documented.

**Comment by LKaemmerling** [comment]
Hey @agnf,

thank you for the report! I fixed it with #97.
